clippy_lints/src/methods/mod.rs

```diff
@@ -2603,6 +2648,112 @@ fn lint_map_flatten<'tcx>(cx: &LateContext<'tcx>, expr: &'tcx hir::Expr<'_>, map
     }
 }
 
+/// lint use of `<fn>_else(simple closure)` for `Option`s and `Result`s that can be
+/// replaced with `<fn>(return value of simple closure)`
+fn lint_lazy_eval<'tcx>(
```

Comment:
Just move this function to its own module like here: https://github.com/rust-lang/rust-clippy/blob/dff7e74b27f8b450c7a084f04ec54a198c0f016f/clippy_lints/src/methods/manual_saturating_arithmetic.rs#L9

Reply:
Sure I'll do that, thanks